libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -382,9 +382,20 @@ float AP_Baro::get_external_temperature(const uint8_t instance) const
     if (_last_external_temperature_ms != 0 && AP_HAL::millis() - _last_external_temperature_ms < 10000) {
         return _external_temperature;
     }
-    // if we don't have an external temperature then use the minimum
-    // of the barometer temperature and 35 degrees C. The reason for
-    // not just using the baro temperature is it tends to read high,
+    
+    // if we don't have an external temperature then try to use temperature
+    // from the airspeed sensor
+    AP_Airspeed *airspeed;
+    if ((airspeed = AP_Airspeed::get_singleton()) != nullptr) {
```

Comment:
please don't put assignment statements in IF statements.

Reply:
Fixed